http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/MockSamDAO.scala

```diff
@@ -105,7 +105,29 @@ class MockSamDAO extends SamDAO[IO] {
   override def getStatus(implicit ev: ApplicativeAsk[IO, TraceId]): IO[StatusCheckResponse] =
     IO.pure(StatusCheckResponse(true, Map.empty))
 
-  override def getListOfResourcePermissions(resource: SamResource, authHeader: Authorization)(implicit ev: ApplicativeAsk[IO, TraceId]): IO[List[String]] = IO.pure(List.empty)
+  override def getListOfResourcePermissions(resource: SamResource, authHeader: Authorization)(
```

Comment:
question on this implementation, is there a reason why we have a pattern match here, but not for the same method in httpsamdao?

Reply:
this is just mock...in httpsamdao, we'd be getting the info from sam.